tests/other_test.go

```diff
@@ -25,6 +28,35 @@ func TestRunBasicExample(t *testing.T) {
 	assert.NotNil(t, output, "Expected some output")
 }
 
+func TestRunAdvanceExample(t *testing.T) {
```

Comment:
I think we should run this in pr_test.go - otherwise we have no test running in PRs thats testing CBRs